docs/spec/reserve-pool/TODO.md

```diff
@@ -0,0 +1,3 @@
+TODO 
+
+The reserve pool is the pool of collected funds for use by governance taken via the `CommunityTax`
```

Comment:
Can we mention that it can't be spent (at the moment)?

Reply:
added `Currently with the SDK, tokens collected by the CommunityTax are accounted for but unspendable.`